Content.Server/GameObjects/Components/Singularity/ContainmentFieldGeneratorComponent.cs

```diff
@@ -198,6 +204,18 @@ public void CollideWith(IEntity collidedWith)
             }
         }
 
+        public void UpdateConnectionLights()
+        {
+            if (_pointLightComponent != null)
+            {
+                bool res = (_connection1 != null) || (_connection2 != null);
+                // hmm, this will dirty things even if the value doesn't change.
+                // thus, is checking this a bad thing?
+                if (_pointLightComponent.Enabled != res)
+                    _pointLightComponent.Enabled = res;
```

Comment:
imo the check should be in the pointlightcomponent's enabled setter, just open a small engine pr for a quickmerge :)

Reply:
https://github.com/space-wizards/RobustToolbox/pull/1507

Reply:
See latest commit